src/main/scala/com/fulcrumgenomics/sv/cmdline/ClpGroups.scala

```diff
@@ -5,10 +5,16 @@ import com.fulcrumgenomics.sopt.cmdline.ClpGroup
 /** Groups for organizing command line programs for display. */
 object ClpGroups {
 
-  class _All extends ClpGroup {
-    override val name: String = "All tools"
-    override val description: String = "All tools."
+  class _BreakpointAndSv extends ClpGroup {
+    override val name: String = "Breakpoint and SV Tools"
+    override val description: String = "Primary tools for calling and transforming breakpoints and SVs."
   }
 
-  final val All = classOf[_All]
+  class _Utilities  extends ClpGroup {
+    override val name: String = "Utility Tools"
+    override val description: String = "Helper tools for working with breakpoint or SV data."
+  }
+
+  final val BreakpointAndSv = classOf[_BreakpointAndSv]
+  final val Utilities = classOf[_Utilities]
```

Comment:
This is a backwards compatible change and make the CLI look like:

<img width="1334" alt="Screenshot 2024-04-10 at 12 38 05 PM" src="https://github.com/fulcrumgenomics/fgsv/assets/4634699/b1bb88ce-4668-44ed-8560-643e36819bac">